### DISCUSSION TITLE
How to load safetensors model?

### DISCUSSION THREAD

**Discussion by aldrinjenson** [AbdBarho/stable-diffusion-webui-docker#383] [Q&A] [answered]
Hi, 
Thanks a lot for the docker version.

I downloaded a custom model from civitai.com . it was a safetensor file.
I did a wget url  of the model and saved it to data/Stable-Diffusions folder in my server along with other ckpt files already present.
However, even when docker restarted and UI refreshed, the model doesn't show up in the web ui.

Any idea how to fix this?

**Comment by aldrinjenson** [comment, marked as answer]
Hi, 
this actually got fixed automatically yesterday :) 
Didn't do anything, but when the system was started yesterday, the model started showing up correctly :)
Thanks!

**Comment by afurkank** [comment]
Hi,
How did you do "wget url"? Could you provide the code please? And where did you put the command for downloading the model weights from civit.ai?

**Comment by aldrinjenson** [comment]
Hey it's been an year. I'll tell from what i remember.

1. First, you go to [civit.ai](https://civit.ai) and search for the model you want to download.
2. Once you find the desired model, click on the download button to start the download.
3. Immediately stop the download, right-click the downloaded file's icon and copy the URL link.

Next, log into your server and navigate to the models folder for Stable Diffusion. This should typically be something like `data/stable-diffusion`.

Once you're in the correct directory, you can use the `wget` command to download the model directly into this folder. Here's a sample command using the copied URL:

```sh
wget [copied_url_link]
```

Replace `[copied_url_link]` with the actual URL you copied earlier.

After the model is downloaded, if the Docker container running Stable Diffusion doesn't automatically detect the new model, you can manually reload the models from the UI. Go to the Stable Diffusion UI, and inside the models drop-down menu, click the reload button. This will update the UI with the new model.
